website/addons/wiki/templates/wiki-bar-modal-help.mako

```diff
@@ -0,0 +1,18 @@
+<div class="modal fade" id="wiki-help-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true"></span>&times;</button>
+                    <h3 class="modal-title">Wiki syntax help</h3>
+                </div>
+                <div class="modal-body">
+                  <p>
+                    Wiki uses the Markdown syntax. This gives you many options but can be very simple as well. To see more information and examples go to markdown <a href="https://daringfireball.net/projects/markdown/">guides.</a>
```

Comment:
Could I ask what osf guides page the link is supposed to go to? Like could you give me the link or information on how to find the link. Googling osf guides, does not return anything useful.